modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -261,6 +271,7 @@
       value   => $publishing_api_bearer_token;
   }
 
+  # TO BE DELETED:
   if $email_address_override_whitelist_only {
```

Comment:
This could be `$email_address_override_allowlist_only` with a similar approach taken for the actual list itself.